iis-wf/iis-wf-primary/src/test/resources/eu/dnetlib/iis/wf/primary/processing/sampledataproducer/oozie_app/workflow.xml

```diff
@@ -482,6 +482,22 @@
                     <name>patentServiceEndpointOpsUriRoot</name>
                     <value>expectedOpsUriRoot</value>
                 </property>
+				<property>
+					<name>affiliation_matching_number_of_emitted_files</name>
+					<value>1</value>
+				</property>
+				<property>
+					<name>documentsclassification_main_number_of_partitions</name>
+					<value>1</value>
+				</property>
+				<property>
+					<name>referenceextraction_patent_main_number_of_emitted_files</name>
+					<value>1</value>
+				</property>
+				<property>
+					<name>webcrawlNumberOfEmittedFiles</name>
```

Comment:
This obviously caught my eye (camel vs snake case) but I guess changing this convention here would force us to change the convention for all the webrawl params and it will become slightly off the scope of this task. So I guess we should leave it as it is and maybe commit some time to align on the convention later on.

Reply:
This seems like the best option for now - changing all the names would be quite messy now.